src/MOI_wrapper.jl

```diff
@@ -271,9 +273,9 @@ function backward!(model::Optimizer, params::Array{String}, dl_dz::Array{Float64
 
     grads = []
     LHS = create_LHS_matrix(z, λ, Q, G, h, A)
-    RHS = [dl_dz'; zeros(neq+nineq,1)]
+    RHS = sparse([dl_dz'; zeros(neq+nineq,1)])
```

Comment:
shouldnt it be a vector rather than a one column matrix?

Reply:
yep, changed it